src/aqua/decider/deal_storage.aqua

```diff
@@ -87,6 +105,7 @@ func remove_deal(spell_id: string, deal_id: string, worker_id: string):
     try:
```

Comment:
I suggest to rename `func remove_deal` to `func remove_deal_storage`, otherwise it's not clear at the call site that this function deals with just KV